### PR TITLE
Rename addAdmin message to scheduleUpgrade

### PR DESCRIPTION
**Pull request by AStox**

### CODE REVIEW

**Review by offerijns — APPROVED** [review]
LGTM!

File: src/Messages.sol [hunk 1]
```diff
@@ -746,15 +746,15 @@ library ConnectorMessages {
         remainingRedeemOrder = uint128(_msg.indexUint(105, 16));
     }
 
-    function formatAddAdmin(address user) internal pure returns (bytes memory) {
-        return abi.encodePacked(uint8(Call.AddAdmin), user);
+    function formatScheduleUpgrade(address user) internal pure returns (bytes memory) {
```
Comment (NunoAlexandre) [inline src/Messages.sol:749]: It would be cool to rename user to contract here in solidity as well 🙏🏻

Comment (AStox) [reply src/Messages.sol:749]: `contract` is a keyword in solidity so that won't work, but how about `upgrade`?

Comment (AStox) [reply src/Messages.sol:749]: I suppose `_contract` would be the most appropriate here

**Pull request merged**